Formula/httpstat.rb

```diff
@@ -0,0 +1,16 @@
+class Httpstat < Formula
+  desc "Curl statistics made simple"
+  homepage "https://github.com/reorx/httpstat"
+  url "https://github.com/reorx/httpstat/archive/v1.1.2.tar.gz"
```

Comment:
Actually the file in this version's tar ball does contain the `#!/usr/bin/env python` shebang, you can download and see :)